obspy/core/util/testing.py

```diff
@@ -113,23 +114,113 @@ def add_doctests(testsuite, module_name):
                 pass
 
 
-def checkForMatplotlibCompareImages():
-    try:
-        # trying to stay inside 80 char line
-        import matplotlib.testing.compare as _compare
-        compare_images = _compare.compare_images  # NOQA @UnusedVariable
-    except:
-        return False
-    # matplotlib's (< 1.2) compare_images() uses PIL internally
-    if getMatplotlibVersion() < [1, 2, 0]:
-        try:
-            import PIL  # NOQA @UnusedImport
-        except ImportError:
-            return False
-    return True
-
-
-HAS_COMPARE_IMAGE = checkForMatplotlibCompareImages()
+def compare_images(expected, actual, tol):
+    """
+    Custom version of :func:`matplotlib.testing.compare.compare_images`.
+    This enable ObsPy to have the same image comparison metrics across
+    matplotlib versions. Futhermore nose is no longer a test dependency of
+    ObsPy.
+
+    Only works with png files in contrast to the matplotlib version.
+
+    Compare two "image" files checking differences within a tolerance.
+
+    Parameters
+    ----------
```

Comment:
This is using the Google style of docstring, I guess. So it'll probably cause a bunch of warnings in Sphinx.